Resources/public/scss/modules/content-tree/_list.item.scss

```diff
@@ -77,7 +77,6 @@ $list-item-height: calculateRem(20px);
         border-radius: calculateRem(4px);
         margin-left: calculateRem(26px);
         transition: background 0.3s $ez-admin-transition, color 0.3s $ez-admin-transition;
-        cursor: pointer;
```

Comment:
Why not pointer anymore?

Reply:
I realised that, it is added by default to buttons.